web/src/components/TaskStatusTaxRegistrationSnackbar.tsx

```diff
@@ -0,0 +1,19 @@
+import { SnackbarAlert } from "@/components/njwds-extended/SnackbarAlert";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { ReactElement } from "react";
+
+interface Props {
+  isOpen: boolean;
+  close: () => void;
+}
+
+export const TaxStatusRegistrationSnackbar = (props: Props): ReactElement => {
```

Comment:
Should the component name and the file name match?

Reply:
Yes...oversight.

Reply:
+ typo.